client/src/Movies/MovieList.js

```diff
@@ -19,11 +20,13 @@ const MovieList = props => {
   }, []);
   
   return (
-    <div className="movie-list">
+    <nav className="movie-list">
       {movies.map(movie => (
-        <MovieDetails key={movie.id} movie={movie} />
+        <NavLink to={`/movies/${movie.id}`}>
+          <MovieDetails key={movie.id} movie={movie} />
+        </NavLink>
       ))}
-    </div>
+    </nav>
   );
 }
 
```

Comment:
Think about breaking The MovieDetails function out into a new file to make its own component.
Nothing about the functionality would change only the imports at the top of your files that interact with it. 

You gain clarity, readability and separation of responsibility among the different components.